src/examples/java/io/nats/examples/NatsJsPushSub.java

```diff
@@ -23,43 +23,45 @@
 /**
  * This example will demonstrate JetStream push subscribing. Run NatsJsPub first to setup message data.
  *
- * Usage: java NatsJsPushSub [server]
+ * Run Notes:
+ * - durable is optional, durable behaves differently, try it by running this twice with durable set
+ * - deliver is optional
+ * - msg_count < 1 will just loop until there are no more messages
+ *
+ * Usage: java NatsJsPushSub [-s server] [-strm stream] [-sub subject] [-dur durable] [-dlvr deliver] [-mcnt msgCount]
  *   Use tls:// or opentls:// to require tls, via the Default SSLContext
  *   Set the environment variable NATS_NKEY to use challenge response authentication by setting a file containing your private key.
  *   Set the environment variable NATS_CREDS to use JWT/NKey authentication by setting a file containing your user creds.
  *   Use the URL for user/pass/token authentication.
  */
 public class NatsJsPushSub {
 
-    // STREAM and SUBJECT are required.
-    // DURABLE is optional (null), durable behaves differently, try it by running this twice with durable set
-    // DELIVER is optional (null)
-    // MSG_COUNT < 1 will just loop until there are no more messages
-    static final String STREAM = "example-stream";
-    static final String SUBJECT = "example-subject";
-    static final String DURABLE = null; // "push-sub-durable";
-    static final String DELIVER = null; // "push-sub-deliver";
-    static final int MSG_COUNT = 0;
-
     public static void main(String[] args) {
-        String server = ExampleArgs.getServer(args);
-        int count = MSG_COUNT < 1 ? Integer.MAX_VALUE : MSG_COUNT;
+        ExampleArgs exArgs = ExampleArgs.builder()
+                .defaultStream("example-stream")
+                .defaultSubject("example-subject")
+//                .defaultDurable("push-sub-durable")
```

Comment:
Did you intend to leave in?

Reply:
yes, in the instructions at top I mention that there is different behavior when durable